receiver/gitproviderreceiver/README.md

```diff
@@ -19,35 +19,52 @@ The current default set of metrics common across all vendors can be found in [do
 
 These default metrics can be used as leading indicators to the DORA metrics; helping provide insight into modern-day engineering practices.
 
-## GitHub Metrics
+## GitHub Scraper
```

Comment:
Nit: It makes more sense to me to have `Getting Started` first, then the `GitHub Scraper` section below the list of scrapers. Personal preference though, your call.

Reply:
Yea, that makes sense!

Reply:
Swapped it around.